src/test/java/pages/AbstractPage.java

```diff
@@ -0,0 +1,30 @@
+package pages;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public abstract class AbstractPage {
+    public static final long WAIT_TIMEOUT_SECONDS = 50;
```

Comment:
protected

Reply:
Refactored, thank you.
Please see in the following commit.